di-ipv-core-stub/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM openjdk:17-jdk-slim@sha256:28ce3bf1f7612d6624b3899937cee7880b88b7740632d002724ddf77a22e37e8 AS build
+WORKDIR /home/gradle/src
+COPY src src
+COPY gradlew .
+COPY gradle gradle
+COPY build.gradle .
+
+RUN ./gradlew build --no-daemon
+
+FROM openjdk:17-jdk-slim@sha256:28ce3bf1f7612d6624b3899937cee7880b88b7740632d002724ddf77a22e37e8
+
+ENV PORT 8085
+ENV DEBUG_PORT 8087
+ENV JAVA_TOOL_OPTIONS -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:$DEBUG_PORT
+RUN addgroup --system --gid 1001 appgroup && adduser --system --uid 1001 appuser --gid 1001
+RUN mkdir /app
+RUN apt-get update && apt-get install -y curl
+COPY --from=build \
+	/home/gradle/src/build/distributions/src.tar \
+	/app/
+
+WORKDIR /app
+ADD config config
```

Comment:
This is adding the contents of `di-ipv-config/stubs/di-ipv-core-stub` into the Docker image.

Are we happy with those files being bundled into the image? 

Ideally I'd like this Docker image to be retrievable via `ghcr.io` and usable with the minimum of fuss - so bundling in the data file does make using this easier - but I thought I should check about data files as its more usual to wire them up separately.